### PR TITLE
Modules controller phpunit tests

### PR DESCRIPTION
**Pull request by didoda**
This provides `ModulesControllerTest` class, to test `ModulesController` class.
No mocks used (as decided by BEdita developers team).

### CODE REVIEW

File: tests/TestCase/Controller/ModulesControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,772 @@
...
+    public function testDescendants() : void
+    {
+        // Setup controller for test
+        $this->setupController();
+
+        // do controller call
+        $result = $this->controller->descendants();
+
+        // verify response status code and type
+        static::assertTrue(is_array($result));
```
Comment (stefanorosanelli) [inline tests/TestCase/Controller/ModulesControllerTest.php]: more meaningful test could be if `$result` is empty (on `documents`) or not empty (on `objects`)

Comment (didoda) [reply tests/TestCase/Controller/ModulesControllerTest.php]: fair enough. coming soon ;)
